packages/canner-graphql-interface/src/connector/localStorageConnector.ts

```diff
@@ -0,0 +1,165 @@
+import { Connector, Pagination } from './types';
+import isEmpty from 'lodash/isEmpty';
+import _last from 'lodash/last';
+import mapValues from 'lodash/mapValues';
+import LocalStorage from 'lowdb/adapters/LocalStorage';
+import low from 'lowdb';
+import uuid from 'uuid/v4';
+import { paginator, filter } from 'canner-graphql-utils';
+import { Field, Types } from 'canner-graphql-utils/lib/schema/types';
+
+const recursivePayload = ({data, schema}: {data: any, schema: Field}) => {
+  return mapValues(data, (val, key) => {
+    const field = schema.getChild(key);
+    const type = field.getType();
+    if (type === Types.ARRAY) {
+      if (isEmpty(val) || isEmpty(val.set)) {
+        return [];
+      }
+
+      return val.set;
+    }
+
+    return val;
+  });
+};
+
+interface Hooks {
+  afterListResolveByUnique?: (key: string, row: any) => void;
+  afterListResolveQuery?: (key: string, rows: any) => void;
+  afterResolveToOne?: (to: string, id: string, row: any) => void;
+}
+
+export default class LocalStorageConnector implements Connector {
+  private db: any;
+  private hooks: Hooks;
+
+  constructor({defaultData, hooks}: {defaultData?: any, hooks?: Hooks}) {
+    const adapter = new LocalStorage();
```

Comment:
@chihsuan,
I think this line should be `const adapter = new LocalStorage(CANNER_LOCAL_STORAGE_KEY)`
see [source](https://github.com/typicode/lowdb/blob/master/src/adapters/LocalStorage.js#L6)

And we can export `CANNER_LOCAL_STORAGE_KEY`, so that in CLI we can call `localStorage.removeItem(CANNER_LOCAL_STORAGE_KEY)` instead of removing all localStorage

Reply:
Great! I just added the `CANNER_LOCAL_STORAGE_KEY` as the public static readonly variable of LocalStorageConnector so we can use 
it like `LocalStorageConnector.CANNER_LOCAL_STORAGE_KEY`. I hope it might be more clear. Thanks.